tests/base/symmetric_tensor_39.cc

```diff
@@ -0,0 +1,69 @@
+#include <iostream>
+#include <fstream>
+
+#include <deal.II/base/symmetric_tensor.h>
+#include <deal.II/base/vectorization.h>
+
+using namespace dealii;
+
+int main()
+{
+  std::string logname = "output";
+  std::ofstream logfile(logname);
+
+#if defined DEAL_II_COMPILER_VECTORIZATION_LEVEL
+  std::cout << "Vectorization preprocessor variable found!" << std::endl;
+#endif
+
+#define INVALID_INT_REPRESENTATION 0
```

Comment:
since i could not find what exactly those define mean, i just got a feeling that they are processor/platform dependent. So it may break things unintentionally.